### ISSUE TITLE
General Error since 2.17.0

### ISSUE THREAD

**Issue by Fr4mby** [WeakAuras/WeakAuras2#2106]
## Describe the bug

WeakAura don't even load since the 2.17.0 update.
I can't open the WeakAura interface, even if some bars are still visible.

Here the error messages i have at the start of the UI:

5x WeakAuras\BuffTrigger2.lua:1817: table index is nil
[string "@WeakAuras\BuffTrigger2.lua"]:1817: in function <WeakAuras\BuffTrigger2.lua:1762>
[string "@WeakAuras\BuffTrigger2.lua"]:1888: in function `LoadDisplays'
[string "@WeakAuras\WeakAuras-2.17.0.lua"]:2312: in function `LoadDisplays'
[string "@WeakAuras\WeakAuras.lua"]:2146: in function <WeakAuras\WeakAuras.lua:2032>
[string "@WeakAuras\WeakAuras.lua"]:2253: in function `ReloadAll'
[string "@WeakAuras\WeakAuras.lua"]:1888: in function `ScanAll'
[string "@WeakAuras\WeakAuras.lua"]:1834: in function `Resume'
[string "@WeakAuras\WeakAuras.lua"]:1652: in function <WeakAuras\WeakAuras.lua:1603>

[string "@WeakAuras\WeakAuras-2.17.0.lua"]:1759: in function <WeakAuras\WeakAuras.lua:1710>

### To Reproduce

Error messages pop at the start of the UI

**Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**

**Which version of WeakAuras are you using?**
2.17.0

**Are you on World of Warcraft Classic or Retail?**
- [ ] Classic
- [X ] Retail

**Was it working in a previous version? If yes, which was the last good one?**
It was working

**Comment by Nikio87** [comment]
I can confirm the same issue for the retail version (pastbin from my buggrabber: https://pastebin.com/t2nagGif)

**Comment by mrbuds** [comment]
> 
> 
> I can confirm the same issue for the retail version (pastbin from my buggrabber: https://pastebin.com/t2nagGif)
> 
> I also found that there seems to be an invisible frame for the menu (found with VDT) but no UI elements for the WA Options Frame are shown.
> 
> Some additional information from me:
> If I remove the WeakAuras.lua file from the Account (WoWDir/_ retail _/WTF/Account/AccountName/SavedVariables ) this issue is not appearing which means that a clear version of WeakAuras works.

if you find an other issue please open a new ticket using the issue template

**Comment by mrbuds** [comment]
@Fr4mby can you give your WeakAuras.lua from savedvariable ?

**Comment by Fr4mby** [comment]
@mrbuds here it is. And thank you for your help!
[WeakAuras.zip](https://github.com/WeakAuras/WeakAuras2/files/4486929/WeakAuras.zip)

**Comment by Fr4mby** [comment]
Ok, i found out that i was an unfinished icon which generated me this error.
I created an icon with a trigger on a specific unit and i let the field of the unit empty.
It seems that previously, it was not considered as an error and now yes (i assume).
Hopefully, i successfully rollbacked on a previous saved version of my weak auras, and i exported as a chain each group of icons.
I imported them back one by one and discovered this.
I hope this will help other people with the same error.
